src/types/lazy.go

```diff
@@ -121,8 +121,12 @@ func wrap(typ byte, byteFields []uint8, args ...string) string {
 	return string(buf)
 }
 
-func unwrap1(s string) (one string) {
-	return s[stringLenBytes+1:] // do not care about length, there is only 1 param
+func unwrap1(s string) string {
+	if len(s) > stringLenBytes+1 {
+		return s[stringLenBytes+1:]
+	}
+
+	return ""
```

Comment:
I don't like the fact that it returns an empty string.
If this is happening, we probably have a problem here

Reply:
Previously, in case of incorrect use there was a panic at the unwrap level. Now panic or an error will occur at the logic level. There should be no problems

Reply:
Most likely we are looking at types before they were wrapped

Reply:
Yes, but in my case I want to try to unwrap an array element. In any case, this logic will stop causing panic at the unwraping level, and will cause an error above. It will be immediately clear, there are no hidden traps here